### PR TITLE
:wrench: Add script to fix touched attributes

### PR DESCRIPTION
**Pull request by hirunatan**

### CODE REVIEW

File: backend/src/app/srepl/fixes.clj [hunk 1]
```diff
@@ -328,3 +329,58 @@
...
+                       (println "Page " (:name page))
+                       (h/update-shapes page (partial update-shape page)))]
+
+     (println "Updating " (:name file) (:id file))
+     (update file :data h/update-pages update-page)))
+
+  ([file save?]
+   (let [file (-> file
+                  (update :data blob/decode)
+                  (fix-component-root))]
```
Comment (pabloalba) [inline backend/src/app/srepl/fixes.clj]: It should be fix-touched

File: backend/src/app/srepl/fixes.clj [hunk 2]
```diff
@@ -328,3 +329,58 @@
...
+                                    (let [equal?
+                                          (if (= group :geometry-group)
+                                            (if (#{:width :height} attr)
+                                              (gsh/close-attrs? attr (get shape attr) (get ref-shape attr) 1)
+                                              true)
+                                            (gsh/close-attrs? attr (get shape attr) (get ref-shape attr)))]
+                                      (when (and (not equal?) (not (cph/touched-group? shape group)))
+                                        (println " -> set touched " (:name shape) (:id shape) attr group))
+                                      (cond-> shape
+                                        (not equal?)
```
Comment (pabloalba) [inline backend/src/app/srepl/fixes.clj]: I think it should be the same that for the println:

(and (not equal?) (not (cph/touched-group? shape group))

File: backend/src/app/srepl/fixes.clj (hunk 1 above)
Comment (hirunatan) [reply backend/src/app/srepl/fixes.clj]: fixed

File: backend/src/app/srepl/fixes.clj (hunk 2 above)
Comment (hirunatan) [reply backend/src/app/srepl/fixes.clj]: It's not relevant, because the update would not have any effect. But you are true, it's better so. Changed.

**Pull request merged**